Cargo.toml

```diff
@@ -18,12 +18,12 @@ keywords = ["async", "fs", "io-uring"]
 
 [dependencies]
 tokio = { version = "1.2", features = ["net", "rt"] }
-scoped-tls = "1.0.0"
 slab = "0.4.2"
 libc = "0.2.80"
 io-uring = { version = "0.5.8", features = ["unstable"] }
 socket2 = { version = "0.4.4", features = ["all"] }
 bytes = { version = "1.0", optional = true }
+once_cell = "1.16.0"
```

Comment:
Why use this crate when tokio::sync::OnceCell exists? I haven't looked at the details. You may very well have a good reason.

Reply:
I just think that the initialization function of OnceCell does not use async, so there is no need to use the async version. Just like Mutex. 
Use tokio::sync::OnceCell can reduce extra dependencies. And it's more tokio. It is also a nice choice. But tokio::sync::OnceCell will add an await dispatch.

I'd love to hear your opinion.

Reply:
I think I understood the reason, but as you've guessed, adding an await for a one time initialization feature isn't a concern, or not nearly as much as introducing a dependency. Extra dependencies for examples, benchmarks and tests aren't nearly the concern but it is something that everyone using the library would give some brain cycles to. I would say it's better to stay in the tokio ecosystem where possible. Your point that even tokio recommends using the std Mutex when it is appropriate I agree with. We don't have an extensive benchmarking suite yet, it's one of the PRs waiting to be merged; not to make it more extensive yet, but to make what's there more useful.

Reply:
Thanks for your patient reply, I'll fix it.